lfucg-exactions/client/js/components/AccountForm.js

```diff
@@ -0,0 +1,272 @@
+                                                <input type="text" className="form-control" placeholder="Phone Number" />
+                                            </FormGroup>
+                                        </div>
+                                        <div className="col-sm-6">
+                                            <FormGroup label="* Email" id="email">
+                                                <input type="email" className="form-control" placeholder="Email" />
+                                            </FormGroup>
+                                        </div>
+                                    </div>
+                                </fieldset>
+                                <button disabled={!submitEnabled} className="btn btn-lex">Submit</button>
+                                {!submitEnabled ? (
+                                    <div>
+                                        <div className="clearfix" />
+                                        <span> * All required fields must be filled.</span>
+                                    </div>
+                                ) : null
+                                }
+                            </form>
+                        </div>
+                    </div>
+                </div>
+
+                <Footer />
+            </div>
+        );
+    }
+}
+
+function mapStateToProps(state) {
+    return {
+        activeForm: state.activeForm,
+        accounts: state.accounts,
+    };
+}
+
+function mapDispatchToProps(dispatch, params) {
+    const selectedAccount = params.params.id;
+
+    return {
+        onComponentDidMount() {
+            dispatch(formInit());
+            dispatch(getMe())
+            .then((data_me) => {
+                if (data_me.error) {
+                    hashHistory.push('login/');
+                }
+                if (selectedAccount) {
+                    dispatch(getAccountID(selectedAccount))
+                    .then((data_account) => {
+                        const update = {
+                            account_name: data_account.response.account_name,
+                            contact_first_name: data_account.response.contact_first_name,
+                            contact_last_name: data_account.response.contact_last_name,
+                            address_city: data_account.response.address_city,
+                            address_state: data_account.response.address_state,
+                            address_zip: data_account.response.address_zip,
+                            phone: data_account.response.phone,
+                            email: data_account.response.email,
+                        };
+                        dispatch(formUpdate(update));
+                    });
+                }
+            });
+        },
+        formChange(field) {
+            return (e, ...args) => {
+                const value = typeof e.target.value !== 'undefined' ? e.target.value : args[1];
```

Comment:
This function works for any field, and can even be called on multiple fields within one component.